### PR TITLE
Migrate CI to GH Actions

### PR DESCRIPTION
**Pull request by ondrajodas**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+name: GitHub Actions
+on: [ push ]
+concurrency: ci-${{ github.ref }} # to avoid tag collisions in the ECR
+env:
+  # Name of the image in the ECR
+  APP_IMAGE: keboola-component
```
Comment (AdamVyborny) [inline .github/workflows/push.yml:6]: `keboola/ssh-keygen`?

**Review by AdamVyborny — COMMENTED** [review]
Ještě badge v readme :)

**Comment by ondrajodas** [comment]
done

File: .github/workflows/push.yml (hunk 1 above)
Comment (ondrajodas) [reply .github/workflows/push.yml]: https://github.com/keboola/ssh-keygen/pull/7/commits/bb9a501001903c9146bc5961c28ea369264fae0a

**Review by AdamVyborny — APPROVED** [review]

**Pull request merged**